packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-type-form.tsx

```diff
@@ -146,194 +194,265 @@ export const PropertyTypeForm = ({
-              clearErrors("name");
-              setTitleValid(false);
+        >
+          <FontAwesomeIcon icon={faClose} />
+        </IconButton>
+      </Box>
+      <Box
+        minWidth={500}
+        p={3}
+        component="form"
+        display="block"
+        onSubmit={handleSubmit}
+      >
+        <Stack
+          alignItems="stretch"
+          spacing={3}
+          sx={{
+            [`.${inputLabelClasses.root}`]: {
+              display: "flex",
+              alignItems: "center",
             },
-            async validate(value) {
-              const propertyTypeId = generateInitialPropertyTypeId(
-                generatePropertyTypeBaseUriForUser(value),
-              );
+          }}
+        >
+          <TextField
+            label="Singular name"
+            required
+            placeholder="e.g. Stock Price"
+            disabled={fieldProps.name?.disabled ?? isSubmitting}
+            {...(!fieldProps.name?.disabled && {
+              error: !!nameError,
+              helperText: nameError?.message,
+              success: titleValid,
+            })}
+            {...register("name", {
+              required: true,
+              onChange() {
+                clearErrors("name");
+                setTitleValid(false);
+              },
+              async validate(value) {
+                const propertyTypeId = generateInitialPropertyTypeId(
+                  generatePropertyTypeBaseUriForUser(value),
+                );
 
-              const res = await getPropertyType({ data: propertyTypeId });
+                const res = await getPropertyType({
+                  data: propertyTypeId,
+                });
 
-              const exists =
-                !res.data || !!getPropertyTypeById(res.data, propertyTypeId);
+                const exists =
+                  !res.data || !!getPropertyTypeById(res.data, propertyTypeId);
 
-              if (getValues("name") === value && !exists) {
-                setTitleValid(true);
-              }
+                if (getValues("name") === value && !exists) {
+                  setTitleValid(true);
+                } else {
+                  setTitleValid(false);
+                }
 
-              return exists ? "Property type name must be unique" : true;
-            },
-          })}
-        />
-        <TextField
-          multiline
-          inputProps={{ minRows: 1 }}
-          label={
-            <>
-              Description{" "}
-              <Tooltip
-                placement="top"
-                title="Descriptions help people understand what property types can be used for, and help make them more discoverable (allowing for reuse)."
-                PopperProps={{
-                  modifiers: [
-                    {
-                      name: "offset",
-                      options: {
-                        offset: [0, 8],
+                return exists ? "Property type name must be unique" : true;
```

Comment:
This prevents us from editing a property, because it already exists, right?

```suggestion
                return exists ? "Property type name must be unique" : true;
```

Reply:
Because slugs are calculated from names, the decision for now is that names can't change

Reply:
What I meant is this is disabling the submit button, so we can't submit the changes to the property type.

Reply:
That isn't my experience. Can you show me a loom?

Reply:
It seems like it's working on the user's workspace, but the submit button is disabled on org namespaces. Is this intended?